src/v/cluster/metadata_dissemination_handler.cc

```diff
@@ -84,13 +84,15 @@ metadata_dissemination_handler::do_update_leadership(
 
 static get_leadership_reply
 make_get_leadership_reply(const partition_leaders_table& leaders) {
-    std::vector<ntp_leader> ret;
+    fragmented_vector<ntp_leader> ret;
+
     leaders.for_each_leader([&ret](
                               model::topic_namespace_view tp_ns,
                               model::partition_id pid,
                               std::optional<model::node_id> leader,
                               model::term_id term) mutable {
-        ret.emplace_back(model::ntp(tp_ns.ns, tp_ns.tp, pid), term, leader);
+        ret.push_back(
```

Comment:
I believe there is an emplace_back in frag vector that Alexey recently added.. if you wish to backport.. guess its fine without it too.

Reply:
> I believe there is an emplace_back in frag vector that Alexey recently added.

Oh, same comment! https://github.com/redpanda-data/redpanda/pull/10620/files#r1188452142